publishing/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
```

Comment:
That's pretty old, and has less than a year of support left. Something newer would be nice?